src/app/gui/item-set-list/item-set-list.component.css

```diff
@@ -1,4 +1,4 @@
-ul.item-set-list {
```

Comment:
Because the component's HTML is pretty simple and only has one `<ul>` element (`item-set-list` is after all just a list), so this simplifies the HTML and CSS a bit.

Reply:
I disagree that "simple" is better here. Unordered List is a semantic element, the CSS a specific styling for _that list_ and a class declaration in HTML really not much of an overhead.

I'm ok with it in this case, but would prefer not mixing backend improvements with subjective simplifications.